packages/cosmic-swingset/x/swingset/handler.go

```diff
@@ -164,7 +164,7 @@ func handleMsgProvision(ctx sdk.Context, keeper Keeper, msg MsgProvision) (*sdk.
 	}
 
 	// Create the account, if it doesn't already exist.
-	egress := types.NewEgress(msg.Nickname, msg.Address)
+	egress := types.NewEgress(msg.Nickname, msg.Address, msg.PowerFlags)
```

Comment:
Am I correct in thinking that the author of the `ProvisionOne` transaction can choose whatever `PowerFlags` they want (i.e. the single "give them everything" one we added, which only actually gives them the ability to set up network connections), but these transactions will only be accepted in messages signed by keys associated with accounts that hold a `provisionpass` token? And that this PR doesn't show the `provisionpass` check because that was already present in the previous revisions?

Reply:
There is no "give them everything" flag, that only exists when running on the fake chain.  You are correct that this is checked against the `provisionpass`, which was already present.